code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -916,6 +916,10 @@ GLOBAL_LIST_EMPTY(PDAs)
 	if(installed_cartridge.charges <=0)
 		balloon_alert(user, "Out of charges")
 		return ..()
+
+	if(target.GetComponent(/datum/component/sound_player))
+		return
```

Comment:
Actually I feel like this should have feedback along the lines of "This is already hacked"

Reply:
```suggestion
	if(target.GetComponent(/datum/component/sound_player))
		balloon_alert(user, "This is already hacked")
		return
```